data/os/Debian/16.04.yaml

```diff
@@ -0,0 +1,4 @@
+---
+timezone::check_hwclock_enabled_cmd: 'grep LOCAL /etc/adjtime'
+timezone::check_hwclock_disabled_cmd: 'grep UTC /etc/adjtime'
```

Comment:
Where have you found that? I've tried it on Ubuntu 16.04. and `/etc/adjtime` looks like:
```
0.0 0 0.0
```